### DISCUSSION TITLE
why miner address is zero address?

### DISCUSSION THREAD

**Discussion by joe2kaleidochain** [0xPolygon/polygon-edge#736] [General]
## [Subject of the issue]
miner address is zero address
### Description
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "parentHash": "0xb7796d585ca17338695d439aca7336742e7961218ae7078e7a07b08f023c8619",
    "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
    **"miner": "0x0000000000000000000000000000000000000000",**
    "stateRoot": "0x3ebb36475288952111a2b9967b94153299d845ce0dc944b91d39185b6eec277d",
    "transactionsRoot": "0x20197f469343fe659f3563d932636257498963261ae92148fcb6513cb0a8f1f1",
    "receiptsRoot": "0x056b23fbba480696b65fe5a59b8f2148a1299103c4f57df839233af2cf4ca2d2",
    "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
    "difficulty": "0x5fa9",
    "totalDifficulty": "0x5fa9",
    "size": "0x4c7",
    "number": "0x5fa9",
    "gasLimit": "0x5f5e100",
    "gasUsed": "0x5208",
    "timestamp": "0x63213f33",
    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000f90230f90120f8469470878322027776c0c474e48de4d0030f4f219c5db0835040373aa7d1df94c8e27390e8bed3d1d813c1ea92061ceaa767e177fd2944ae31650ff29325fbc797ee463513f86bf8469468b4cc33aac25cf969fb4ac30e5c253791725b60b0b6181a80f97cf20fde909ea9405d639e054e601b06f2d6452d5bb63c9d79747e00d2986ff48416a14efd6f73339e58a5f8469430901fd8c800ace0ed7733ede4ff43b0278317bdb0a83fe4ffeef5a055405f883316331786bcb736224c6d32fcc5d0ea337da62e42f43a22aa6b7d10ab6643b0a7144cf6d6f84694acf65ff41a5fa354d4020aad6f4c758699efa2edb0a8892e22f5fadab5e4683331bc0a12c14cb61fb3bc8ad5bb073e47d2e8dbafa29c85beb3237ebc8a840272a0d56e7510b841071dec00c5123316359df216ff917d87baa1fff40c0e1606367ef2132eab73d50213464e44c1ebacbea65f593da826f9dccbf02366cb02843417a36d5b10a7c800f8630eb860acb7f59ed1eee436ae58d0aa1bcdb33d41c42a23ef8e780063958a5c74491ebf82a84c762a2b4ddabba7fc90c44f29f903e2c6b9c690228dfa3a9c492fd543447c5d8bcd60e35ca1992c9e506470af8db928061efb09836755a97f289177246bf86307b86085e68a409942f18fa347e926f31dc4b553e3a37cc001d7e88464543c918a57db65a4e4c28cc11a34fbb19043034dde5d0b57dcdb1aa7fe03591658ebd04e0abe4ac55d26cf8fa75505f04dd29cdd74e8cf127d04789b68fda0944e77135e0f62",
    "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365",
    "nonce": "0x0000000000000000",
    "hash": "0x14266b7976f19cc4b086aec5eda7a2b13ca68d54c41c946fd490979501b0ac28",
    "transactions": [
      "0x9cfdadca62b7ccdd3febab1802d06c75e671df43fb62261ac3dd1b9973588524"
    ],
    "uncles": []
  }
}
Describe your issue in as much detail as possible here

### Your environment

* linux
* development
* branch that causes this issue

### Steps to reproduce

* Tell us how to reproduce this issue <br />
* Where the issue is, if you know <br />
* Which commands triggered the issue, if any

### Expected behaviour

Tell us what should happen

### Actual behaviour

Tell us what happens instead

### Logs

Please paste any logs here that demonstrate the issue, if they exist

### Proposed solution

If you have an idea of how to fix this issue, please write it down here, so we can begin discussing it

**Comment by laviniat1996** [comment]
Hi @joe2kaleidochain ,

I have converted the issue you opened into a discussion.

We are aware that that the miner address is zero, and there is an explanation for this. The `miner ` field is used only to propose a vote for a certain address, so it isn't the actual address of the proposer. You can find a more detailed reply in our docs [here](https://docs.polygon.technology/docs/edge/faq/validators#why-do-the-json-rpc-commands-eth_getblockbynumber-and-eth_getblockbyhash-not-return-the-miners-address).
